docs/docs/starlark-reference/post-http-request-recipe.md

```diff
@@ -16,18 +16,18 @@ post_request_recipe = PostHttpRequestRecipe(
     endpoint = "/endpoint",
 
     # The content type header of the request (e.g. application/json, text/plain, etc)
-    # MANDATORY
```

Comment:
Not sure why those were advertised with MANDATORY. They were effectively optional

Reply:
Done!